src/pages/about.js

```diff
@@ -7,11 +7,83 @@ const AboutPage = ({data, location}) => {
+          <p>Our goals for this website are to help people working on this project and our immediate peers and neighbors to:</p>
+          <ul>
+            <li>Understand what has to happen to decarbonize each US state, and what policies need to be passed to drive those actions</li>
+            <li>Understand the highest impact ways individuals can help by</li>
+            <li>Electrifying their own fossil fuel infrastructure</li>
+            <li>Joining political campaigns to pass those policies</li> 
+            <li>Dispel common misconceptions and strawman arguments against electrification</li>
+          </ul>
+          <h2 className="pt-3">Team and credits</h2>
+          <p>This is an all-volunteer project built at <a href="https://chihacknight.org/">Chi Hack Night</a>, a weekly civic tech and data event based in Chicago.</p>
+          <p>The team includes: <a href="https://derekeder.com">Derek Eder</a>,<a href="https://www.linkedin.com/in/jpvelez/"> Juan-Pablo Velez</a>, <a href="https://viktorkoves.com/">Viktor Köves</a>, <a href="https://chihacknight.org/board-of-directors/sean-watland.html">Sean Watland</a>, <a href="https://www.linkedin.com/in/hkier/">Howard Kier</a>, <a href="https://dylanhalpern.com/">Dylan Halpern</a>, <a href="https://www.elisarudolph.com/">Elisa Rudolph</a>,<a href="https://www.linkedin.com/in/joyce-ohiri/"> Joyce Ohiri</a>, <a href="https://www.linkedin.com/in/sjegoodman/">Samantha Goodman</a>, and <a href="https://www.linkedin.com/in/shelbybarron/">Shelby Barron</a>.</p>
+          <p>The initial code for this project was forked from <a href="https://greenjobsdata.com/">Who benefits from climate ambition?</a> by Data for Progress, Sunrise Movement and DataMade.</p>
+          
+          <h2 className="pt-3">Data and sources</h2>
+          <p>All of the data used on this website are from publicly available and trusted government and nonprofit sources. Below is a description of the primary datasets and their publishers. For more details on how we discovered and worked with this data, take a look at our <a href="https://github.com/chihacknight/decarbonize-my-state/issues?q=label%3Adata">GitHub issue tracker for data</a> and our <a href="https://github.com/chihacknight/decarbonize-my-state/tree/main/data">data folder</a> of this open source project.</p>
+          
+          <h3 className="pt-2"><strong>US Emissions by State</strong></h3>
```

Comment:
Why is this heading bolded and the others aren't? I'd make these consistent but do `<h3 className="pt-2 font-weight-bold">`